notebook/static-src/base/js/namespace.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+"use strict";
+
+if (!window.Jupyter) {
+    var Jupyter = {};
+    Jupyter.version = "4.1.0.dev";
```

Comment:
I don't even know how to do that from node.  A few minutes of Googling didn't get me a good answer ... Searched for things like "find node npm package version programmatically".